cirrus/cirrus-qa.sh

```diff
@@ -6,7 +6,8 @@ source set_maven_build_version $BUILD_NUMBER
 
 # Get the Jenkins plugin HPI file. We will need it to install our plugin.
 wget --user $ARTIFACTORY_DEPLOY_USERNAME --password $ARTIFACTORY_DEPLOY_PASSWORD $ARTIFACTORY_URL/sonarsource-public-qa/org/jenkins-ci/plugins/sonar/$NEW_VERSION/sonar-$NEW_VERSION.hpi
-LOCAL_JARS=`pwd`/sonar-$CURRENT_VERSION.hpi
+export LOCAL_JARS=`pwd`/sonar-$NEW_VERSION.hpi
+echo "LOCAL_JARS: $LOCAL_JARS"
```

Comment:
You want to keep it, or was it for debugging purposes? I think it is okay to have it, just asking.

Reply:
Yes I want to keep it, it can help if we have the same kind of issue in the future